docs/README.md

```diff
@@ -20,10 +20,11 @@ deploy functions, applications, or containers workloads.
 
 ## Components
 
-Knative consists of the Serving and Eventing components:
+Knative consists of the Serving, Eventing and Client (CLI) components:
```

Comment:
The client isn't a new component of Knative; it's a tool for managing the existing components of Knative. This section will remain unchanged. I'd recommend instead adding an H2 under this, `## Client (Kn)`